task1/server.js

```diff
@@ -13,61 +17,57 @@ http
       time: new Date()
     };
 
-    if (fs.existsSync(path)) {
-      writeToJson(path, requestInfo);
-    } else {
-      createNewFile(path, requestInfo);
+    try {
+      await writeToJson(path, requestInfo);
+    }
+    catch(err) {
+      console.error(err);
+      return;
```

Comment:
its not good, because, if we do this, user will wait until response,  till request timeout